apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -181,8 +181,10 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         state?.organization?.organizationType);
     const currentOrganization: OrganizationResponseInterface = useSelector((state: AppState) =>
         state.organization.organization);
+    const isFapiCompliantApp: boolean = initialValues?.isFAPIApplication;
```

Comment:
Lets use destructuring here so that we can extract out any further properties in the future more cleanly if required.
```suggestion
    const { isFapiCompliantApp } = initialValues;
```

Reply:
Applied with af36774